.github/workflows/tcr.yml

```diff
@@ -7,41 +7,24 @@ on:
 
 jobs:
   test:
-    name: Test
+    name: "Test: Ruby ${{ matrix.ruby }}"
     runs-on: ubuntu-latest
-
     strategy:
+      fail-fast: false
       matrix:
         ruby:
-          - "2.0.0"
-          - "2.1.9"
-          - "2.2.10"
-          - "2.3.7"
-          - "2.4.4"
-          - "2.5.1"
-          - "2.7.3"
-          - "3.0.1"
-
+          - "2.7"
+          - "3.0"
+          - "3.1"
     steps:
       - uses: actions/checkout@v2
-
-      - name: Initial setup
+      - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-
-      - name: Install dependencies
-        run: bundle install
-
-      - name: Run tests
+          # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+      - name: Ruby Version
+        run: ruby --version
+      - name: Run Tests
         run: bundle exec rspec
-
-  testall:
```

Comment:
This stub test is to make branch protections easier to configure. Let's keep it.